libraries/Microsoft.Bot.Builder.AI.LUIS/V3/LuisPredictionOptions.cs

```diff
@@ -32,6 +33,7 @@ public class LuisPredictionOptions
         /// </summary>
         /// <value>True to include API results.</value>
         /// <remarks>This is mainly useful for testing or getting access to LUIS features not yet in the SDK.</remarks>
+        [Obsolete]
         public bool IncludeAPIResults { get; set; } = false;
```

Comment:
Why is this obsolete?

Reply:
I moved all the common functionality of the recognizer to the abstract class LuisRecognizerOptions, and left the endpoint options as prediction options